apis/Google.Storage.V1/Google.Storage.V1.IntegrationTests/UrlSignerTest.cs

```diff
@@ -40,6 +42,25 @@ public UrlSignerTest(StorageFixture fixture)
             _fixture.RegisterDelayTests(this);
         }
 
+        /// <summary>
+        /// Gets the name of the delay test associated with the specified delay test initialization method.
+        /// </summary>
+        private static string GetDelayTestName([CallerMemberName] string initMethodName = null)
+        {
+            Debug.Assert(initMethodName.EndsWith(StorageFixture.DelayTestSuffix),
```

Comment:
Either CheckArgument (for the first one) or CheckState (for the others). Whatever makes the most sense, although it frankly doesn't matter too much, so long as it goes bang :)